### ISSUE TITLE
Doesn't seem to save settings

### ISSUE THREAD

**Issue by davestewart** [anlutro/laravel-settings#36]
Just installed this on L5 and I get a settings file which looks like this: `{}` when running your example code and an array that looks like this: `Array()`

I followed your setup instructions to the letter.

**Comment by anlutro** [comment]
Did you add the middleware? I don't use Laravel 5 myself so I can't really verify that it works in production.

If you manually call `Settings::save()` after setting the values, does that update the file?

**Comment by davestewart** [comment]
Hey there.

I tried multiple combinations of middleware, save, etc.

I've since installed another package that is working for me, just thought I would let you know.

It might be the docs, it might be Laravel 5, not sure!

**Comment by bweston92** [comment]
Confirmed working for `4.2` and `5.0`, `5.1` and `5.2`

**Comment by drtzack** [comment]
I've similiar problem.

json store works fine for me but database doesn't work

When I save It's not checking is the record exists. It just creating so many records.

Also there is a problem with cache or something else.

In my first try I used "dc_factor" as key but then I changed it to "factor" but it was still creating "dc_factor" I've just searched in everywhere but it wasn't in anywere. Cleared cache but still same but When I changed store database to json. Then json to database again. Its worked fine. But still It can't check that is record exists in datase. It creates same data to forever.
